src/components/calcite-pick-list/calcite-pick-list.tsx

```diff
@@ -199,6 +202,30 @@ export class CalcitePickList {
     });
   }
 
+  handleFilter = (event) => {
+    const filteredData = event.detail;
+    const values = filteredData.map((item) => item.value);
+    this.items.forEach((item) => {
+      if (values.indexOf(item.value) === -1) {
+        item.setAttribute("hidden", "");
+      } else {
+        item.removeAttribute("hidden");
+      }
+    });
+  };
+
+  getItemData() {
+    const result = [];
+    this.items.forEach((item) => {
+      const obj = {};
+      Array.from(item.attributes).forEach((attr: any) => {
```

Comment:
Are you seeing any warnings/errors by not typing `attr`? TS should infer the `Attr` type.

Reply:
No. I can drop the any here.